color-string.js

```diff
@@ -84,11 +84,11 @@ function getHsla(string) {
    if (!string) {
       return;
    }
-   var hsl = /^hsla?\(\s*([+-]?\d+)(?:deg)?\s*,\s*([+-]?[\d\.]+)%\s*,\s*([+-]?[\d\.]+)%\s*(?:,\s*([+-]?[\d\.]+)\s*)?\)/;
+   var hsl = /^hsla?\(\s*([+-]?\d+(?:\.\d*)?)(?:deg)?\s*,\s*([+-]?[\d\.]+)%\s*,\s*([+-]?[\d\.]+)%\s*(?:,\s*([+-]?[\d\.]+)\s*)?\)/;
```

Comment:
It most likely wouldn't handle such numbers. I can modify the regex to make it similar to the ones being used for the other parameters like `[\d\.]+`. That of course brings in its own flaws.
Alternatively I could use a regex like `\d*[\.]?\d+` which would still not be able to handle exponent notations.
I will update the PR with the second option, as it seems to cover more cases.